README.md

```diff
@@ -192,6 +192,22 @@ Command Options
 | `--solution=$solution`            | Set Visual Studio Solution version (win)
 
 
+Configuration
+--------
+
+#### `node-gyp` responds to environment variables:
+Variables take the form `npm_config_OPTION_NAME` for any of the options
+listed above.<br>
+They can be set in two ways:
+1. as `npm` configuration - these work only when `node-gyp` is triggered by
+ `npm`. for example:<br>
+ `$ npm config set [--global] npm_config_devdir /tmp/.gyp`
+2. as environment variables - these work also when `node-gyp` is invoked
+ directly:<br>
+ `> set npm_config_devdir=c:\temp\.gyp` for Windows or<br>
+ `$ export npm_config_devdir=/tmp/.gyp` for unix like environments
```

Comment:
A few remarks:

- is the `####` at the top necessary?
- are the `<br>` tags necessary?
- maybe s/triggered/executed/?  'triggered' is somewhat unspecific.
- s/for example/For example/
- s/unix like/UNIX-like/
- insert a line or two explaining dashes should be replaced with underscores